packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js

```diff
@@ -118,6 +154,10 @@ export default class QuanticSearchBox extends LightningElement {
       'quantic__selectsuggestion',
       this.selectSuggestion
     );
+    this.removeEventListener(
```

Comment:
You gave me the same comment on my pr recently :D 

If you check for `if (!this.disableRecentQuerySuggestions)` to add the event listener, why don't you also check to remove the event listener? :D

Reply:
I felt like removing wouldn't harm, if it's there it will be removed, if it's not, then it's not there, trying to remove wouldn't cause a problem.